pyvista/plotting/camera.py

```diff
@@ -31,6 +31,38 @@ def __init__(self):
         self._elevation = 0.0
         self._azimuth = 0.0
 
+    def __repr__(self):
+       """Print a repr specifying the id of the camera and its camera type."""
+       return (f'<{self.__class__.__name__} at {hex(id(self))}>')
+
+    def __eq__(self, other):
+        """Compare whether the relevant attributes of two camera are equal."""
+        # attributes which are native python types and thus implement __eq__
```

Comment:
Do we have non-native types? We should check all attributes.

And small nitpick: at least `model_transform_matrix` seems to be numpy array, which isn't a native python type :)

Reply:
You are right. It is not native python type :disappointed: . I will check non-native type element :)